vagrant_repo/data_bags/cerner_splunk/cluster-vagrant.json

```diff
@@ -8,7 +8,7 @@
   "replication_ports": {
     "8080" : { }
   },
-  "license_uri": null,
+  "license_uri": "https://33.33.33.30:8089",
```

Comment:
Similar to [the comment above](https://github.com/cerner/cerner_splunk/pull/108/files#r58477945) let's leave this as null with instructions in the docs to replace it.